go/roothash/api/commitment/executor.go

```diff
@@ -46,6 +46,9 @@ type ComputeResultsHeader struct {
 	IORoot       *hash.Hash `json:"io_root,omitempty"`
 	StateRoot    *hash.Hash `json:"state_root,omitempty"`
 	MessagesHash *hash.Hash `json:"messages_hash,omitempty"`
+	// TODO: consensus breaking change. Open a follow up issue to include this
+	// for a later release.
+	// EventsHash   *hash.Hash `json:"events_hash,omitempty"`
```

Comment:
It would probably make sense to include this in `ProposedBatch` as well so that the workers can quickly check whether it matches with their local view before processing the batch.